.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  env: {
+    jest: true,
```

Comment:
related to file name: since we have .prettierrc shouldn't we have have .eslintrc instead of .eslintrc.js (or the other way around) ?